irb2400_client.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
I the functionality in this script can be accomplished by means of a launch file then that'd be preferable.